### PR TITLE
Remove '--cluster-conf' cli option and its references

### PR DESCRIPTION
**Pull request by shylesh**
Signed-off-by: Shylesh Kumar <shmohan@redhat.com>

### CODE REVIEW

File: conf/examples/telemeter_stage.yaml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 ---
 ENV_DATA:
-  # for testing of OCS telemetry integration with stage instance of telemeter
-  # use via `--cluster-conf conf/ocsci/stage_telemeter.yaml`
```
Comment (vasukulkarni) [inline conf/examples/telemeter_stage.yaml]: I think we need this but instead it should be --ocsci-conf

File: docs/usage.md [hunk 2]
```diff
@@ -56,10 +56,6 @@ to the pytest.
     [default config](https://github.com/red-hat-storage/ocs-ci/tree/master/ocs_ci/framework/conf/default_config.yaml).
     This is the repeatable parameter and you can use it multiple times for
     adding multiple config files. (The last one overwrites previous config!)
-* `--cluster-conf` - with this configuration you can overwrite the default
```
Comment (vasukulkarni) [inline docs/usage.md]: only --cluster-conf should change to --ocsici-conf, the rest of the help message should exist.

Comment (petr-balogh) [reply docs/usage.md]: This parameter is described few lines above:  https://github.com/red-hat-storage/ocs-ci/pull/1259/files#diff-4be2078a125fe364f7198675d51bc688L54

File: conf/examples/telemeter_stage.yaml (hunk 1 above)
Comment (shylesh) [reply conf/examples/telemeter_stage.yaml]: @vasukulkarni @petr-balogh , is it same for monitoring.yaml as well. 
```--ocsci-conf conf/ocsci/monitoring.yaml``` and ```--ocsci-conf conf/ocsci/stage_telemeter.yaml```

File: docs/usage.md [hunk 3]
```diff
@@ -98,13 +94,11 @@ that can be used to access an existing cluster.
 If you would like to deploy new cluster you can run following command:
 ```bash
 run-ci -m deployment --ocsci-conf conf/ocsci/custom_config.yaml \
-    --cluster-conf conf/ocs_basic_install.yml \
```
Comment (petr-balogh) [inline docs/usage.md]: --ocsci-conf conf/path_to_config_to_be_used.yaml

File: docs/usage.md (hunk 2 above)
Comment (shylesh) [reply docs/usage.md]: Done

**Comment by shylesh** [comment]
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3759/

**Comment by shylesh** [comment]
Verified plain ipi deployment

**Pull request merged**